tests/frontends/test_pytorch.py

```diff
@@ -225,7 +225,8 @@ def step(model, inp, target):
 
     loss, grad = step(model, inp, target)
 
-    assert loss == 42.759910583496094
+    # assert loss == 42.759910583496094
```

Comment:
```suggestion
```

Also remove the other commented out asserts.

Reply:
Done !